src/Administration/Resources/app/administration/src/module/sw-mail-template/page/sw-mail-template-index/index.js

```diff
@@ -1,25 +1,30 @@
 import template from './sw-mail-template-index.html.twig';
 
-const { Component, Mixin } = Shopware;
```

Comment:
this is a breaking change. Removing a mixin can also remove props, methods, data and so on. This leads to failures in plugins working with this component.